### PR TITLE
Add test cases for org.apache.synapse.commons.executors.config

### PR DESCRIPTION
**Pull request by manoramahp**
## Purpose
Unit tests to cover the functionality in org.apache.synapse.commons.executors.config package

### CODE REVIEW

File: modules/commons/src/test/java/org/apache/synapse/commons/executors/config/PriorityExecutorFactoryTest.java [hunk 1]
```diff
@@ -0,0 +1,207 @@
...
+import junit.framework.TestCase;
+import org.apache.axiom.om.OMElement;
+import org.apache.axiom.om.util.AXIOMUtil;
+import org.apache.axis2.AxisFault;
+import org.apache.synapse.commons.executors.PriorityExecutor;
+
+import javax.xml.stream.XMLStreamException;
+import java.util.Properties;
+
+public class PriorityExecutorFactoryTest extends TestCase {
```
Comment (milindaperera) [inline modules/commons/src/test/java/org/apache/synapse/commons/executors/config/PriorityExecutorFactoryTest.java]: class javadoc comment missing

File: modules/commons/src/test/java/org/apache/synapse/commons/executors/config/PriorityExecutorFactoryTest.java [hunk 2]
```diff
@@ -0,0 +1,207 @@
...
+    public void testCreateExecutor() throws XMLStreamException, AxisFault {
+        OMElement element = AXIOMUtil.stringToOM("<priority-executor xmlns:ns=\"http://ws.apache.org/ns/synapse\" name=\"executor1\">" +
+                "<ns:queues></ns:queues>" +
+                "</priority-executor>");
+        PriorityExecutor executor = PriorityExecutorFactory.createExecutor("http://ws.apache.org/ns/synapse",
+                element, false, new Properties());
+        assertNotNull("Error creating PriorityExecutor", executor);
+    }
+
+    public void testCreateExecutorWithQueues() throws XMLStreamException, AxisFault {
```
Comment (milindaperera) [inline modules/commons/src/test/java/org/apache/synapse/commons/executors/config/PriorityExecutorFactoryTest.java]: shall we add small description Javadoc mentioning what is tested in each testcase

File: modules/commons/src/test/java/org/apache/synapse/commons/executors/config/PriorityExecutorSerializerTest.java [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+import org.apache.synapse.commons.executors.PriorityExecutor;
+import org.apache.synapse.commons.executors.PRRNextQueueAlgorithm;
+import org.apache.synapse.commons.executors.queues.FixedSizeQueue;
+import org.apache.synapse.commons.executors.queues.UnboundedQueue;
+
+import javax.xml.stream.XMLStreamException;
+import java.util.ArrayList;
+import java.util.List;
+
+public class PriorityExecutorSerializerTest extends TestCase {
```
Comment (milindaperera) [inline modules/commons/src/test/java/org/apache/synapse/commons/executors/config/PriorityExecutorSerializerTest.java]: javadoc comment

File: modules/commons/src/test/java/org/apache/synapse/commons/executors/config/PriorityExecutorFactoryTest.java (hunk 1 above)
Comment (manoramahp) [reply modules/commons/src/test/java/org/apache/synapse/commons/executors/config/PriorityExecutorFactoryTest.java]: Added

File: modules/commons/src/test/java/org/apache/synapse/commons/executors/config/PriorityExecutorFactoryTest.java (hunk 2 above)
Comment (manoramahp) [reply modules/commons/src/test/java/org/apache/synapse/commons/executors/config/PriorityExecutorFactoryTest.java]: Added

File: modules/commons/src/test/java/org/apache/synapse/commons/executors/config/PriorityExecutorSerializerTest.java (hunk 3 above)
Comment (manoramahp) [reply modules/commons/src/test/java/org/apache/synapse/commons/executors/config/PriorityExecutorSerializerTest.java]: Added

**Pull request merged**